app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js

```diff
@@ -48,11 +47,10 @@ export const getColumns = (content, contentFields, { orderedContentFields, defec
   }));
 
   orderedData.forEach((item) => {
-    contentFields.forEach((field) => {
-      if (field === item.key) {
-        columns.push([field, item.value]);
-      }
-    });
+    const findedItem = contentFields.find((field) => field === item.key);
```

Comment:
`foundItem`? Or maybe it's better to give it more descriptive name?

Reply:
Yes, of course! My mistake